src/ndv/viewer/_backends/_vispy.py

```diff
@@ -85,36 +87,7 @@ def cursor_at(self, pos: Sequence[float]) -> Qt.CursorShape | None:
         return self._cursor_at(self.pos)
 
 
-class EditableROI(scene.visuals.Polygon):
-    """Base Class defining behavior all editable ROIs should have."""
```

Comment:
i do like the forward thinking concept... but as I was reviewing, and while trying to orient myself to all of the various protocols and subclasses, I found myself reminding myself a couple of time "oh yeah, that EditableROI thing is really only the rectangular ROI"... so rather than making a do-nothing mixin that only one class uses, let's just make a note about how this could be expanded in the future